_data/universities/lmu.yml

```diff
@@ -0,0 +1,5 @@
+name: "[Loyola Marymount University](https://www.lmu.edu/coronavirus/communitymessages/messagefromthepresidentonlineinstruction/)"
```

Comment:
Would you mind renaming the file to `loyola-marymount.yml` or something similar? I'd like to avoid initial overload if possible.